examples/native-hybrid-app/android/app/build.gradle

```diff
@@ -11,7 +11,7 @@ plugins {
 
 android {
     namespace 'com.datadoghq.hybrid_flutter_example'
-    compileSdk 33
+    compileSdk 34
```

Comment:
I noticed that `compileSdk` version is upgraded to 34, but `targetSdk` stays at 32, is that wanted?

Reply:
It should be fine, especially in example apps. In reality I may want to switch all the apps to use `flutter.targetSdkVersion` in the future but that can be a separate change.